src/lib/netlist/solver/nld_ms_w.h

```diff
@@ -289,43 +303,43 @@ namespace netlist::solver
 					if (H[i][i] == plib::constants<float_type>::zero())
 						plib::perrlogger("{} H singular\n", this->name());
 					const float_type f = plib::reciprocal(H[i][i]);
-					for (unsigned j = i+1; j < row_count; j++)
+					for (unsigned j = i + 1; j < row_count; j++)
 					{
-						const float_type f1 = - f * H[j][i];
+						const float_type f1 = -f * H[j][i];
 
 						// FIXME: comparison to zero
 						if (f1 != plib::constants<float_type>::zero())
 						{
-							float_type *pj = &H[j][i+1];
-							const float_type *pi = &H[i][i+1];
-							for (unsigned k = 0; k < row_count-i-1; k++)
+							float_type       *pj = &H[j][i + 1];
+							const float_type *pi = &H[i][i + 1];
+							for (unsigned k = 0; k < row_count - i - 1; k++)
 								pj[k] += f1 * pi[k];
-								//H[j][k] += f1 * H[i][k];
+							// H[j][k] += f1 * H[i][k];
 							w[j] += f1 * w[i];
 						}
 					}
 				}
 				// Back substitution
-				//inv(H) w = t     w = H t
-				for (unsigned j = row_count; j-- > 0; )
+				// inv(H) w = t     w = H t
+				for (unsigned j = row_count; j-- > 0;)
 				{
-					float_type tmp = 0;
-					const float_type *pj = &H[j][j+1];
-					const float_type *tj = &t[j+1];
-					for (unsigned k = 0; k < row_count-j-1; k++)
+					float_type        tmp = 0;
+					const float_type *pj = &H[j][j + 1];
+					const float_type *tj = &t[j + 1];
+					for (unsigned k = 0; k < row_count - j - 1; k++)
 						tmp += pj[k] * tj[k];
-						//tmp += H[j][k] * t[k];
+					// tmp += H[j][k] * t[k];
```

Comment:
Another case where the commented statement belongs in the loop body logically, and it doesn’t make sense after being reformatted.

Reply:
Will fix this.